File: soal9.pas
program Soal9;
uses crt;
var
    berat, tinggi, beratIdeal, selisih: real;
begin
    clrscr;
    write('Masukkan berat badan (kg): ');
    readln(berat);
    write('Masukkan tinggi badan (cm): ');
    readln(tinggi);

    beratIdeal := tinggi - 100;
    selisih := berat - beratIdeal;
    if selisih < 0 then
        selisih := -selisih;

    if selisih <= 2 then
        writeln('Ideal')
    else
        writeln('Tidak ideal');

    readln;
end.

File: soal9.c
#include <stdio.h>

int main() {
    float berat, tinggi, beratIdeal, selisih;

    printf("Masukkan berat badan (kg): ");
    scanf("%f", &berat);
    printf("Masukkan tinggi badan (cm): ");
    scanf("%f", &tinggi);

    beratIdeal = tinggi - 100;
    selisih = berat - beratIdeal;
    if (selisih < 0)
        selisih = -selisih;

    if (selisih <= 2)
        printf("Ideal\n");
    else
        printf("Tidak ideal\n");

    return 0;
}